### ISSUE TITLE
Improve error handling

### ISSUE THREAD

**Issue by jstet** [CorrelAid/correlaid_website#24]

**Comment by KonradUdoHannes** [comment]
Currently missing fields on the CMS side can cause the site not to function at all.

As a concrete example we had the case that `ctas` on `navbar.home` where missing a text entry, which caused the top level +layout.svelte to crash on parsing. We should
- [ ] make parsing errors explicit (instead of cryptic "undefined does not have property ..." and similar)
- [ ] allow these parsing errors to be logged or displayed. This potentially affects dynamic ssr runtime issues or static page build time issues.
- [ ] potentially ignore faulty cms configurations to render the rest of the page (i.e. catch parse errors -> log them -> continue without component if possible)

**Comment by jstet** [comment]
connected to #23

**Comment by jstet** [comment]
- [ ] non existing slugs should result in 404

see https://correlaid-website.vercel.app/daten_nutzen/projekte/test/

**Comment by KonradUdoHannes** [comment]
After #295 the only components for which explicit parsing is not done yet are `Html`, `Link_Button`,  `TextContainer` and `Hero`. The use of these components is a little more generic and it might be difficult to fit all into the same pattern as the previously refactored components. After an initial analysis, any parsing of these components should be moved to a separate task if it has to be treated differently in a substantial way.

**Comment by KonradUdoHannes** [comment]
The previous comment should now refer to #296, because there was a merge issue (wrong branch) and #295 was thrown away post merge.

**Comment by jstet** [comment]
I would say error handling was improved and we have or can open sep. issues for further improvements
